File: 2/C/Grace.c
/*
	Grace
*/
#include <stdio.h>
#include <fcntl.h>
#define STR "/*%c%cGrace%c*/%c#include <stdio.h>%c#include <fcntl.h>%c#define STR %c%s%c%c#define PRINT(fd) dprintf(fd, STR, 10, 9, 10, 10, 10, 10, 34, STR, 34, 10, 10, 34, 34, 10, 10)%c#define RUN int main() { int fd = open(%cGrace_kid.c%c, O_CREAT | O_WRONLY, 0644); PRINT(fd); }%cRUN%c"
#define PRINT(fd) dprintf(fd, STR, 10, 9, 10, 10, 10, 10, 34, STR, 34, 10, 10, 34, 34, 10, 10)
#define RUN int main() { int fd = open("Grace_kid.c", O_CREAT | O_WRONLY, 0644); PRINT(fd); }
RUN

File: PYTHON/Grace.py
# Grace
STR = "# Grace%cSTR = %c%s%c%cPRINT = %cf.write(STR %c (10, 34, STR, 34, 10, 34, 37, 34, 10, 34, 34, 10, 10))%c%cRUN = %cf = open('Grace_kid.py', 'w'); exec(PRINT); f.close()%c%cexec(RUN)%c"
PRINT = "f.write(STR % (10, 34, STR, 34, 10, 34, 37, 34, 10, 34, 34, 10, 10))"
RUN = "f = open('Grace_kid.py', 'w'); exec(PRINT); f.close()"
exec(RUN)
